CMakeLists.txt

```diff
@@ -1,22 +1,58 @@
-PROJECT( twbt )
+cmake_minimum_required(VERSION 3.22)
+SET(CMAKE_VERBOSE_MAKEFILE ON)
+SET(DFHACK ../dfhack)
+SET(DFHACK_BUILD ${DFHACK}/build)
+
+IF(EXISTS "${DFHACK_BUILD}/CMakeCache.txt")
+    LOAD_CACHE("${DFHACK_BUILD}" READ_WITH_PREFIX DEFAULT_ CMAKE_INSTALL_PREFIX)
+    SET(CMAKE_INSTALL_PREFIX ${DEFAULT_CMAKE_INSTALL_PREFIX})
```

Comment:
This is redundant for me, because I build dfhack and twbt side by side and so this file already sees this variable correctly. What is it's benefit?